docs/api/developer.md

```diff
@@ -255,6 +255,26 @@ classes allow for new model classes to be easily integrated with the module.
    autotune.Tunable
 ```
 
+## Model hub
```

Comment:
I think everything should live in the user side of the docs. If you want you can add two sentences here with a link to the user side.